invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides

```diff
@@ -55,10 +55,19 @@
   padding: 0;
 }
 
-.ui.button.fluid-mobile {
-  @media all and (max-width: @largestMobileScreen) {
-    width: 100%;
-    min-width: 0em;
-    display: block;
+.ui.button{
+  &.fluid-tablet {
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+      width: 100%;
+      min-width: 0em;
+      display: block;
+    }
+  }
+  &.fluid-mobile {
+    @media all and (max-width: @largestMobileScreen) {
+      width: 100%;
+      min-width: 0em;
```

Comment:
I think I commented on that, but my comment was removed? 
I would apply min-width at least of 4em, 0 will cause the dropdowns to be very varied between them in length